packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -644,6 +644,137 @@ export async function closeCookieModals(page: Page): Promise<void> {
     await page.evaluate(getCookieClosingScript());
 }
 
+interface HandleCloudflareChallengeOptions {
+    verbose?: boolean;
+    sleepSecs?: number;
+    clickCallback?: (page: Page, boundingBox: DOMRect) => Promise<void>;
+    isChallengeCallback?: (page: Page) => Promise<boolean>;
+    isBlockedCallback?: (page: Page) => Promise<boolean>;
```

Comment:
Can we get some one-line comments? While `(most | all)` of those are quite intuitive, it might be overwhelming for users unfamiliar with solving Cloudflare challenges (e.g. why is `clickCallback` being passed the bounding box, difference between challenge and blocked etc.).